pkg/netutil/netutil.go

```diff
@@ -45,12 +47,13 @@ type CNIEnv struct {
 }
 
 func DefaultConfigList(e *CNIEnv) (*NetworkConfigList, error) {
-	return GenerateConfigList(e, DefaultID, DefaultNetworkName, DefaultCIDR)
+	return GenerateConfigList(e, nil, DefaultID, DefaultNetworkName, DefaultCIDR)
 }
 
 type ConfigListTemplateOpts struct {
 	ID           int
 	Name         string // e.g. "nerdctl"
+	Labels       string // e.g. `,{"group":"bigger"}`
```

Comment:
What does bigger mean?

Reply:
it was just an example, I changed to a more meaningful example